### ISSUE TITLE
Canary Version 0.0.5 is incompatible with Ars Nouveau whirlisprig

### ISSUE THREAD

**Issue by Zelioth** [AbdElAziz333/Canary#23]
Canary Version 0.0.5 On my published custom mod pack Friends United when will get to the world generation screen then cause stop mid way. One time crashing while generating but every other time it stalls at 75% and gives errors in the log but does not crash. I will try to create and publish logs. 

Minecraft 1.19.2
Forge 43.1.43
Canary Current version working 0.0.4
Canary Updated version not working 0.0.5

I am testing an update for my pack fixing a few bugs and adding a few mods. This issue with all the mods updated and added does not happen with canary 0.0.4, only 0.0.5

https://pastebin.com/7qg2eUYX

**Comment by AbdElAziz333** [comment]
oo, other one told me about this issue, i will look into that.

**Comment by adriancpp** [comment]
I have similiar problem,
I used Canary 0.0.4 on both client and server, and it works fine, but today
i updated Canary to 0.0.5 and client seems working good, but server stopped while starting:

```
[09:05:42] [main/ERROR] [ne.mi.fm.lo.RuntimeDistCleaner/DISTXFORM]: Attempted to load class net/minecraft/client/gui/Gui for invalid dist DEDICATED_SERVER
[09:05:44] [main/ERROR] [ne.mi.fm.lo.RuntimeDistCleaner/DISTXFORM]: Attempted to load class net/minecraft/client/renderer/LevelRenderer for invalid dist DEDICATED_SERVER
```

minecraft 1.19.2
forge 43.1.43

**Comment by AbdElAziz333** [comment]
Its incompatbility with Ars Nouveau Whirlisprig, i will add a fix soon.

**Comment by AbdElAziz333** [comment]
fixed in 0.0.6 update.
